### PR TITLE
Update components pf4

### PR DESCRIPTION
**Pull request by Hyperkid123**
This is a first batch of updates for PF4v4 migration.

**Note that builds for other packages might fail due to the dependencies upgrade**. I think its a better idea to upgrade packages one by one that making one humongous PR with every change. It is easier to review and find possible issues.

###Notable changes
- FilterChip
  - chip groups has changed API: https://www.patternfly.org/v4/documentation/react/overview/upgrade-guide#chip-group
  - ChipGroupToolbarItem was moved to ChipGroup component according to PF guideline
- PrimaryToolbar
  - DataToolbar component no longer exits
  - It was replaced by Toolbar component
- Selects
  - `isExpanded` prop no longer exists
  - replaced by `isOpen`

Snapshots are currently busted because very large number of PF components are now represented as Unknown or Component is virtual DOM

### Upgrade deadlines

|1. | ci-beta on 7/6 |
|-- | -- |
|2. | prod-beta on 7/13 |
|3. | ci-stable by 7/20 |
|4. | prod-stable on 7/30|

### CODE REVIEW

File: packages/components/src/Components/InvalidObject/InvalidObject.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ import { withRouter } from 'react-router-dom';
 const InvalidObject = ({ history, ...props }) => {
     return (
         <section {...props} className="pf-l-page__main-section pf-c-page__main-section ins-c-component_invalid-componet">
-            <Title size='3xl'>This doesn&apos;t seem to exist.</Title>
+            <Title headingLevel="h2" size='3xl'>This doesn&apos;t seem to exist.</Title>
```
Comment (karelhala) [inline packages/components/src/Components/InvalidObject/InvalidObject.js:13]: This should be `headingLevel="h1"`

Comment (Hyperkid123) [reply packages/components/src/Components/InvalidObject/InvalidObject.js:13]: Should it? Regarding the docs, the default value used to be `h2`: https://www.patternfly.org/v4/documentation/react/overview/upgrade-guide#title

Comment (karelhala) [reply packages/components/src/Components/InvalidObject/InvalidObject.js:13]: Check the snapshots it used to be `h1`, looks like the docs are lying...

Comment (Hyperkid123) [reply packages/components/src/Components/InvalidObject/InvalidObject.js:13]: Well the docs says `h2`, but the code says `h1`. Nice to know that I can't relly on the upgrade guide.

https://github.com/patternfly/patternfly-react/pull/3922/files#diff-29db5c83c2f983f2d91ed0f5ffaf2817L31

**Comment by rvsia** [comment]
Is it a good idea to work on it when the issue with components names in virtual dom is not resolved? I understand it can takes months... but this update make all the snapshots basically useless and unreliable. :crying_cat_face:

File: packages/components/src/Components/InvalidObject/__snapshots__/InvalidObject.test.js.snap [hunk 2]
```diff
@@ -83,15 +83,16 @@ exports[`InvalidObject component should render 1`] = `
               }
             }
           >
-            <Title
+            <Component
+              headingLevel="h2"
               size="3xl"
             >
-              <h1
+              <h2
```
Comment (karelhala) [inline packages/components/src/Components/InvalidObject/__snapshots__/InvalidObject.test.js.snap:90]: Looks like the snapshots were not updated.

**Comment by Hyperkid123** [comment]
@rvsia yup I agree. Unfortunately, we have a deadline and it has to be upgraded.

Can we rely on PF to release a fix in a near future (By near I mean by the end of this week)?

**Comment by Hyperkid123** [comment]
@karelhala as expected the tests busted (not just snapshots) for other packages. Do you think we can merge it into the staging PF4-update branch and fix the packages one by one?

**Pull request merged**